src/lib/y2storage/planned/partitions_distribution.rb

```diff
@@ -96,7 +96,7 @@ def gaps_count
 
       # Total space available for the planned partitions
       # @return [DiskSize]
-      def spaces_total_disk_size
+      def spaces_total_size
```

Comment:
well, when you renaming it, why not remove `_total_`? for me `gaps_size` and `spaces_size`.

Reply:
Because `disk_size` has proven to be confusing in the past. And `total` has never confused anyone and I personally find it useful for clarification purposes.